yb-voyager/cmd/analyzeSchema.go

```diff
@@ -560,7 +564,16 @@ func checkDDL(sqlInfoArr []sqlInfo, fpath string) {
 			objType := strings.ToUpper(strings.Split(fileName, ".")[0])
 			reportCase(fpath, `temporary table is not a supported clause for drop`,
 				"https://github.com/yugabyte/yb-voyager/issues/705", `remove "temporary" and change it to "drop table"`, objType, sqlInfo.objName, sqlInfo.formattedStmt)
+		} else if regMatch := anydataRegex.FindStringSubmatch(sqlInfo.stmt); regMatch != nil {
+			reportCase(fpath, "AnyData datatype doesn't have a mapping in YugabyteDB", "", `remove the column with AnyData datatype`, "TABLE", regMatch[2], sqlInfo.formattedStmt)
```

Comment:
Can we give the suggestion as an option between Remove the column with AnyData datatype or Change it to a supported datatype. It will be up to the user to take a call.
Same for all instances.